app/src/main/res/values/strings.xml

```diff
@@ -4,84 +4,82 @@
 
     <!--Status-->
     <string name="install">Install</string>
-    <string name="installing">Installing…</string>
-    <string name="reinstalling">Reinstalling…</string>
-    <string name="denied">Install aborted, retry?</string>
+    <string name="installing">Installing</string>
+    <string name="reinstalling">Reinstalling</string>
+    <string name="denied">The installation was aborted by the user</string>
 
-    <string name="reinstallRequired">Re-installation required!</string>
+    <string name="reinstallRequired">Re-installation required</string>
 
     <string name="update">Update</string>
-    <string name="updating">Updating…</string>
+    <string name="updating">Updating</string>
     <string name="updated">Updated</string>
-    <string name="alreadyUpToDate">Already up to date!</string>
+    <string name="alreadyUpToDate">Already up to date</string>
 
-    <string name="uninstall">Uninstall</string>
-    <string name="uninstalling">Uninstalling…</string>
+    <string name="uninstalling">Uninstalling</string>
 
-    <string name="processing">Processing…</string>
-    <string name="downloading">Downloading…</string>
-    <string name="downloaded">Downloaded</string>
+    <string name="processing">Processing</string>
+    <string name="downloading">Downloading</string>
     <string name="failed">Failed</string>
-    <string name="failedRetry">A failure has occurred, retry?</string>
-    <string name="pending_install">Pending…</string>
+    <string name="failedRetry">A failure has occurred</string>
+    <string name="pending_install">Pending</string>
 
-    <string name="syncingMetadata">Syncing metadata</string>
-    <string name="syncUnfinished">Sync has not yet finished</string>
+    <string name="syncingMetadata">Metadata sync is in progress</string>
+    <string name="syncUnfinished">Sync is still in progress</string>
 
     <!--DownloadCallBack.kt-->
-    <string name="DownloadedSuccessfully">Downloaded successfully</string>
-    <string name="dfSecurityError">Download failed due to a security error</string>
-    <string name="dfUnknownHostError">Download failed due to being unable to locate host</string>
-    <string name="dfIoError">Download failed due to an IO error</string>
+    <string name="DownloadedSuccessfully">Download successful</string>
+    <string name="dfSecurityError">Download failed due to a security error. Please retry downloading</string>
+    <string name="dfUnknownHostError">Download failed due to improper DNS configuration. Check DNS/Private DNS settings</string>
```

Comment:
On line 40: `Check DNS/Private DNS settings and try again`

Missing `and try again` here